modules/ztp-configuring-ptp-fast-events.adoc

```diff
@@ -74,7 +73,7 @@ For example, add the following to `group-du-sno-ranGen.yaml`:
 +
 [source,yaml]
 ----
-- fileName: StorageLV.yaml
+- fileName: StorageLV.yaml <1>
```

Comment:
Since we are just documenting events here, we can remove the `example-storage-class-1` and `example-storage-class-2` since they are not required for events.
```
- fileName: StorageLV.yaml 
  policyName: "config-policy"
  spec:
    storageClassDevices:
      - storageClassName: "storage-class-http-events" 
        volumeMode: Filesystem
        fsType: xfs
        devicePaths:
          - /dev/disk/by-partlabel/httpevent1
          - /dev/disk/by-partlabel/httpevent2       
```